lib/rollup/index.js

```diff
@@ -15,64 +13,31 @@ const { minify } = require('terser')
 const plugin = (modulePath, moduleName) => async (files, metalsmith, done) => {
   modulePath = normalize(modulePath)
 
-  // Metalsmith file paths
-  const file = modulePath.replace(/\.mjs$/, '.js')
-  const map = `${file}.map`
+  // Remove Metalsmith file source
+  delete files[modulePath]
 
   try {
-    // Create Rollup bundle
-    const bundle = await rollup({
-      input: join(metalsmith.source(), modulePath),
-      plugins: [
-        nodeResolve(),
-        commonjs()
-      ]
-    })
-
-    // Compile Rollup bundle
-    const result = await bundle.generate({
-      file,
-      sourcemapFile: file,
-      sourcemap: true,
-
-      // Self-executing function, suitable for <script> tags
-      format: 'iife',
-
-      // Used to set the `window` global for 'iife' and 'umd' bundles
-      name: moduleName
-    })
-
-    // Minify Rollup bundle
-    const minified = await minify({ [modulePath]: result.code }, {
-      format: { comments: false },
-
-      // Include source maps
-      sourceMap: {
-        content: result.map,
-        filename: result.map.file,
-        url: `${result.map.file}.map`,
-        includeSources: true
-      },
-
-      // Compatibility workarounds
-      ecma: 5,
-      safari10: true
-    })
-
-    // Update Metalsmith file contents
-    files[file] = {
-      contents: Buffer.from(minified.code)
-    }
-
-    // Add Metalsmith source map file
-    files[map] = {
-      contents: Buffer.from(minified.map)
-    }
-
-    // Remove source module from output when
-    // file path changes (e.g. `.mjs` to `.js`)
-    if (modulePath !== file) {
-      delete files[modulePath]
+    const input = join(metalsmith.source(), modulePath)
+
+    // Create Rollup bundle(s)
+    for (const options of config) {
```

Comment:
Are we likely to need the ability to create multiple bundles in this repo in the future?

If not, could we lose the outer array and this loop and just build based on the single config?

(Suspect this wouldn't be an option if we decide to use `loadConfigFile` as I imagine it'll normalise to an array of config objects anyway?)

Reply:
Yeah spot on, I did this entirely to match the format `loadConfigFile()` returns

Must be the favoured syntax by the Rollup team